boards/arm/nucleo_f767zi/nucleo_f767zi.yaml

```diff
@@ -0,0 +1,25 @@
+identifier: nucleo_f767zi
+name: NUCLEO-F767ZI
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 512
+flash: 2048
+supported:
+  - arduino_i2c
+  - arduino_gpio
+  - arduino_spi
+  - uart
+  - gpio
+  - netif:eth
+  - usb_device
+  - i2c
+  - pwm
+  - spi
+  - watchdog
+  - counter
+  - can
+  - adc
```

Comment:
Board is not supported by `tests/drivers/adc/adc_api` so please remove this line for now.
This is what is preventing shippable (CI) to pass.

Reply:
Okay, done, hope this time it'll pass. I also did the "Xi" change.
Thanks.